python/ray/_private/runtime_env/uv_test.py

```diff
@@ -0,0 +1,29 @@
+from ray._private.runtime_env import uv
+
+import unittest
+from unittest.mock import AsyncMock
+
+
+class TestRuntimeEnv:
+    def uv_config(self):
+        return {"packages": ["requests"]}
+
+    def env_vars(self):
+        return {}
+
+
+class UvProcessorTest(unittest.IsolatedAsyncioTestCase):
+    async def test_run(self):
+        target_dir = "/tmp"
+        runtime_env = TestRuntimeEnv()
+
+        aync_mock = AsyncMock(return_value=None)
+        uv_processor = uv.UvProcessor(target_dir=target_dir, runtime_env=runtime_env)
+
+        with unittest.mock.patch("uv.UvProcessor._install_uv", aync_mock):
+            with unittest.mock.patch("uv.UvProcessor._install_uv_packages", aync_mock):
+                await uv_processor._run()
+
+
+if __name__ == "__main__":
+    unittest.main()
```

Comment:
Let's run using pytest like other tests.

We can have the discussion of moving away from pytest separately

Reply:
The issue is:
- We don't have `pytest` checked in as third party, which means for [hermetic](https://bazel.build/basics/hermeticity) test which bazel encourages, we have to checkin all dependent packages
- In another way, I think `unittest` is strictly better than `pytest`:
  + `pytest` is third party library, while `unittest` built-in one, generally official packages are more well maintained
  + we use `assert` to check correctness in `pytest`, which doesn't print out value if mismatch, while functions like `assertEqual` do, which is provided by `unittest` and it helps debugging